lua/prettier/options.lua

```diff
@@ -117,11 +117,12 @@ function M.setup(user_options)
     return
   end
 
-  user_options = u.tbl_flatten(user_options, should_flatten)
```

Comment:
Let's just add the fallback here and keep everything else the same?

```lua
  user_options = u.tbl_flatten(user_options or {}, should_flatten)
```

Reply:
Updated.